src/Files.App/UserControls/AddressToolbar.xaml

```diff
@@ -27,6 +27,23 @@
 			<converters:NullToTrueConverter x:Key="NullToFalseConverter" Inverse="True" />
 			<converters1:BoolNegationConverter x:Key="BoolNegationConverter" />
 
+			<ResourceDictionary.ThemeDictionaries>
```

Comment:
Should there be a brush for high contrast?

Reply:
Ideally. but it's temporary button so... Idk